testing/terraform/ec2/main.tf

```diff
@@ -81,8 +89,10 @@ resource "null_resource" "main_service_setup" {
 
   provisioner "remote-exec" {
     inline = [
-      # Install Java 11 and tmux
-      "yes | sudo amazon-linux-extras install java-openjdk11",
+      # Install wget
+      "yes | sudo yum install wget",
+      # Install Java 11
+      "sudo yum install java-11-amazon-corretto -y",
```

Comment:
minor: you can install both in one line

```
sudo yum install wget java-11-amazon-corretto -y
```

Reply:
Done and tested here: https://github.com/harrryr/aws-otel-java-instrumentation/actions/runs/7495714707